core/internal/runbranch/resume.go

```diff
@@ -258,6 +258,10 @@ func processResponse(
 		params.Tags = data.GetTags()
 	}
 
+	if params.Notes == "" && data.GetNotes() != nil {
+		params.Notes = *data.GetNotes()
+	}
```

Comment:
Could you please add a quick test to `tests/system_tests/test_core/test_resume.py`?